src/ServiceControlInstaller.Engine/Instances/ServiceControlAuditPersisters.cs

```diff
@@ -31,5 +31,22 @@ public static IReadOnlyList<PersistenceManifest> LoadAllManifests(string zipFile
                 return manifests;
             }
         }
+
+        public static PersistenceManifest GetPersistence(string zipFilePath, string name)
+        {
+            var manifests = LoadAllManifests(zipFilePath);
+
+            if (string.IsNullOrEmpty(name))
+            {
+                // Must always remain RavenDB35 so that SCMU understands that an instance with no configured value is an old Raven 3.5 instance
+                return manifests.Single(m => m.Name == "RavenDB35");
+            }
+
+            return manifests.FirstOrDefault(m => m.Name == name || m.TypeName == name) ?? new PersistenceManifest
```

Comment:
Be explicit in the string comparisons?

ps. Didn't check the using! Drive by code suggestion!

```suggestion
            return manifests.FirstOrDefault(m => m.Name.Equals(name, StringComparison.OrdinalIgnoreCase) || m.TypeName.Equals(name, StringComparison.OrdinalIgnoreCase)) ?? new PersistenceManifest
```